admin/services/pupil.service.js

```diff
@@ -103,4 +103,28 @@ pupilService.sortByStatus = (pupilsList, sortDirection) => {
   })
 }
 
+/**
+ * Sort pupil's array by group.
+ * @param pupilsList
+ * @param sortDirection
+ * @returns {*}
+ */
+pupilService.sortByGroup = (pupilsList, sortDirection) => {
```

Comment:
Why doesn't this sort happen in the database with nulls on top?

Reply:
@jon-shipley please have a look at how this array is been created, 'groups' and 'status' (outcome) are mapped after the initial query.

Reply:
Nulls on top is not the desired behavior.